lagunita/lms/templates/static_templates/copyright.html

```diff
@@ -9,16 +9,18 @@
 <%namespace name='static' file='/static_content.html'/>
 <%namespace name='theme_settings' file='/theme-settings.html'/>
 
-<%block name="pagetitle">${_('Copyright Complaints')}</%block>
-
 <%
   theme_vars = theme_settings.get_theme_vars()
   effective_date = theme_vars['effective_date']
   contact_email = '<a href="mailto:{email}">{email}</a>'.format(email=settings.CONTACT_EMAIL)
 %>
 
+<%block name="pagetitle">${_('Copyright Complaints')}</%block>
+
 <main id="main" aria-label="Content" tabindex="-1">
     <section class="container copyright">
+        <%include file="menu_tos.html" args="current_page='copyright'" />
```

Comment:
menu is imported from a separate file so as to not duplicate code, but the current page is passed in to label the active link